prime-router/src/test/kotlin/cli/helpers/HL7DiffHelperTests.kt

```diff
@@ -92,60 +92,18 @@ class HL7DiffHelperTests {
+OBR|1||232270000212^ProPhase Diagnostics^2.16.840.1.114222.4.1.238646^ISO|55454-3^Hemoglobin A1C ^LN|||20230815164300-0500|||1234567890^Chriscoe^Matthew||||||1528068368^Israel^Rosa^^^^^^&2.16.840.1.113883.19.4.6&ISO^^^^NPI^^^^^^^^MD||||||20230815164519-0500|||F|||||||||&Chriscoe&Matthew||||||||||||||||||||||||||"""
+    private val msgMSH8Empty = """MSH|^~\&#|ProPhase^2.16.840.1.114222.4.1.238646^ISO|ProPhase^33D2215033^CLIA|CDC Prime^2.16.840.1.114222.4.1.237821^ISO|CDC Prime^2.16.840.1.114222.4.1.237821^ISO|20230816123358-0500||ORU^R01^ORU_R01|20230816123358|P|2.5.1|||||||||PHLabReport-NoAck^ELR_Receiver^2.16.840.1.113883.9.11^ISO|
 SFT|Orchard|9.0|Orchard Enterprise|9.0.211217.220208||20220411
 PID|1||0008115-23-02^^^PROPHASE DIAGNOSTICS&2.16.840.1.114222.4.1.238646&ISO^PI^PROPHASE DIAGNOSTICS&2.16.840.1.114222.4.1.238646&ISO||Test^Male||19701031|M||2131-1^Other Race^HL70005|1234 Adams place^^New York^NY^10457||^^^noemail@prophasedx.com^^^|||||||||U^Unknown^HL70189^^^^2.5.1||||||||N||||||
-SPM|1|^232270000212&&2.16.840.1.114222.4.1.238646&ISO||258500001^Nasopharyngeal swab^SCT||||71836000^Nasopharyngeal structure (body structure)^SCT|||||||||20230815164300-0500|20230815164500-0500"""
-    private val MsgSegSFTRemoved = """MSH|^~\&#|ProPhase^2.16.840.1.114222.4.1.238646^ISO|ProPhase^33D2215033^CLIA|CDC Prime^2.16.840.1.114222.4.1.237821^ISO|CDC Prime^2.16.840.1.114222.4.1.237821^ISO|20230816123358-0500||ORU^R01^ORU_R01|20230816123358|P|2.5.1|||||||||PHLabReport-NoAck^ELR_Receiver^2.16.840.1.113883.9.11^ISO|
-PID|1||0008115-23-02^^^PROPHASE DIAGNOSTICS&2.16.840.1.114222.4.1.238646&ISO^PI^PROPHASE DIAGNOSTICS&2.16.840.1.114222.4.1.238646&ISO||Test^Male||19701031|M||2131-1^Other Race^HL70005|1234 Adams place^^New York^NY^10457||^^^noemail@prophasedx.com^^^|||||||||U^Unknown^HL70189^^^^2.5.1||||||||N||||||
-SPM|1|^232270000212&&2.16.840.1.114222.4.1.238646&ISO||258500001^Nasopharyngeal swab^SCT||||71836000^Nasopharyngeal structure (body structure)^SCT|||||||||20230815164300-0500|20230815164500-0500"""
+ORC|RE||232270000212^ProPhase Diagnostics^2.16.840.1.114222.4.1.238646^ISO|^^^|||||20230815164300-0500|1234567890^Chriscoe^Matthew||1528068368^Israel^Rosa^^^^^^&2.16.840.1.113883.19.4.6&ISO^^^^NPI^^^^^^^^MD|^^^ProPhase Labs NY Clinical&CL||20230815164300-0500||||||ProPhase Diagnostics|711 Stewart Ave Ste 200^^Garden City^NY^11530|^^PH^^1^866^7522837|711 Stewart Ave Ste 200^^Garden City^NY^11530
+OBR|1||232270000212^ProPhase Diagnostics^2.16.840.1.114222.4.1.238646^ISO|55454-3^Hemoglobin A1C ^LN|||20230815164300-0500|||1234567890^Chriscoe^Matthew||||||1528068368^Israel^Rosa^^^^^^&2.16.840.1.113883.19.4.6&ISO^^^^NPI^^^^^^^^MD||||||20230815164519-0500|||F|||||||||&Chriscoe&Matthew||||||||||||||||||||||||||"""
 
+// Difference between messages at MSH(1).8.1 Differences: Input had more repeating types for ST, input has 1 and output has 0
```

Comment:
Did you mean to leave this comment here?

Reply:
will remove

Reply:
"Did you decide not to do this, or was it just missing from your commit? If you decided against it, that's fine, but a comment about your decision here would help me know that it wasn't a miss." 
will address this....
thanks for the good comments!